1753683755292-30b3431f487b4cc1863e57a81d78e289.sh

```
@@ -0,0 +1,204 @@
+
+    case "$platform" in
+        Linux|Darwin)
+            log_info "Installing Node.js on $platform..."
+
+            # 安装 nvm
+            log_info "Installing nvm ($NVM_VERSION)..."
+            curl -s https://raw.githubusercontent.com/nvm-sh/nvm/"$NVM_VERSION"/install.sh | bash
+
+            # 加载 nvm
+            log_info "Loading nvm environment..."
+            \. "$HOME/.nvm/nvm.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./.nvm/nvm.sh was not specified as input (see shellcheck -x).

(SC1091)